src/js/media-time-range.js

```diff
@@ -766,7 +769,7 @@ class MediaTimeRange extends MediaChromeRange {
       return;
     }
 
-    const duration = this.mediaDuration;
+    const duration = this.mediaSeekableEnd;
```

Comment:
prefer `mediaSeekableEnd` here to make sure we're not early bailing for DVR cases.